packages/controls/src/icon-radio-group.ts

```diff
@@ -1,47 +0,0 @@
-export const unstable_IconRadioGroupIcon = {
```

Comment:
Moved to https://github.com/makeswift/makeswift/pull/781/files#diff-eabb08fb171c3a089a3f5d3823857c5521aa07b0957a21211f32abc73875d3b4